src/sql/workbench/contrib/notebook/browser/cellViews/outputArea.component.html

```diff
@@ -6,7 +6,7 @@
 -->
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div #outputarea link-handler [isTrusted]="isTrusted" [notebookUri]="notebookUri" class="notebook-output" style="flex: 0 0 auto;">
-		<output-component *ngFor="let output of cellModel.outputs" [cellOutput]="output" [trustedMode] = "cellModel.trustedMode" [cellModel]="cellModel" [activeCellId]="activeCellId" [batchId]="output.metadata?.resultSet?.batchId" [id]="output.metadata?.resultSet?.id">
+		<output-component *ngFor="let output of cellModel.outputs" [cellOutput]="output" [trustedMode] = "cellModel.trustedMode" [cellModel]="cellModel" [activeCellId]="activeCellId" [batchId]="output.metadata?.batchId" [id]="output.metadata?.id">
```

Comment:
Do we still need the batchid/id serialized itself? Once we open a notebook up again we don't care about these id's, right? Can we just keep this info in memory instead?

Reply:
Yeah we don't use it once the notebook is opened up again but keeping it in memory would require some more changes in CellModel (a mapping between outputs and ids, and updating the map each time this._outputs is updated). Since batchid and id are pretty small is it okay if we serialize those?

Reply:
Do they even make sense to serialize? They aren't going to have any meaning outside of the particular run that you did right?

Reply:
Hm yeah that is true - let me make those CellModel changes.